### PR TITLE
feat: b prime full presentation flow working

### PR DESCRIPTION
**Pull request by berendsliedrecht**
Signed-off-by: Berend Sliedrecht <sliedrecht@berend.io>

### CODE REVIEW

File: apps/easypid/src/constants.ts [hunk 1]
```diff
@@ -3,9 +3,8 @@ export const EASYPID_WALLET_PID_PIN_KEY_ID = 'EASYPID_WALLET_PID_PIN_KEY_ID_NO_B
...
------END CERTIFICATE-----`
+export const bdrPidIssuerCertificate =
+  'MIICeTCCAiCgAwIBAgIUB5E9QVZtmUYcDtCjKB/H3VQv72gwCgYIKoZIzj0EAwIwgYgxCzAJBgNVBAYTAkRFMQ8wDQYDVQQHDAZCZXJsaW4xHTAbBgNVBAoMFEJ1bmRlc2RydWNrZXJlaSBHbWJIMREwDwYDVQQLDAhUIENTIElERTE2MDQGA1UEAwwtU1BSSU5EIEZ1bmtlIEVVREkgV2FsbGV0IFByb3RvdHlwZSBJc3N1aW5nIENBMB4XDTI0MDUzMTA2NDgwOVoXDTM0MDUyOTA2NDgwOVowgYgxCzAJBgNVBAYTAkRFMQ8wDQYDVQQHDAZCZXJsaW4xHTAbBgNVBAoMFEJ1bmRlc2RydWNrZXJlaSBHbWJIMREwDwYDVQQLDAhUIENTIElERTE2MDQGA1UEAwwtU1BSSU5EIEZ1bmtlIEVVREkgV2FsbGV0IFByb3RvdHlwZSBJc3N1aW5nIENBMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEYGzdwFDnc7+Kn5ibAvCOM8ke77VQxqfMcwZL8IaIA+WCROcCfmY/giH92qMru5p/kyOivE0RC/IbdMONvDoUyaNmMGQwHQYDVR0OBBYEFNRWGMCJOOgOWIQYyXZiv6u7xZC+MB8GA1UdIwQYMBaAFNRWGMCJOOgOWIQYyXZiv6u7xZC+MBIGA1UdEwEB/wQIMAYBAf8CAQAwDgYDVR0PAQH/BAQDAgGGMAoGCCqGSM49BAMCA0cAMEQCIGEm7wkZKHt/atb4MdFnXW6yrnwMUT2u136gdtl10Y6hAiBuTFqvVYth1rbxzCP0xWZHmQK9kVyxn8GPfX27EIzzsw=='
```
Comment (TimoGlastra) [inline apps/easypid/src/constants.ts:7]: is this just different encoding?

File: apps/easypid/src/constants.ts [hunk 2]
```diff
@@ -18,6 +17,9 @@ export const trustedX509Certificates = [
...
   animoFunkeRelyingPartyCertificate,
   ubiqueRootCertificate,
+  'MIIBDDCBtKADAgECAhBYoGsxg/PbsXjPhRMmMQ7YMAoGCCqGSM49BAMCMAAwHhcNNzAwMTAxMDAwMDAwWhcNMjUxMTIyMDgyMjEyWjAAMDkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDIgAC3A9V8ynqRcVjADqlfpZ9X8mwbew0TuQldH/QOpkadsWjMDAuMCwGA1UdEQQlMCOCITJlM2YtODktMjA1LTEyOC0zNS5uZ3Jvay1mcmVlLmFwcDAKBggqhkjOPQQDAgNHADBEAiB+UkItFHFn6SsDApT2w7SK4vAW/uv94kVMTVjhSR0B1gIgbE7knz8EHfjPU/gvkp7KgAF2fPf0adKMjMEzIXV5n8g=',
+  'MIIBDDCBtKADAgECAhAd+OL1QwkDMec9AAn061uFMAoGCCqGSM49BAMCMAAwHhcNNzAwMTAxMDAwMDAwWhcNMjUxMTIyMDgyMjEyWjAAMDkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDIgAC3A9V8ynqRcVjADqlfpZ9X8mwbew0TuQldH/QOpkadsWjMDAuMCwGA1UdEQQlMCOCITJlM2YtODktMjA1LTEyOC0zNS5uZ3Jvay1mcmVlLmFwcDAKBggqhkjOPQQDAgNHADBEAiBDCH2pxb47I0F3p0oOib1BwurfPx/rZZJQfMAPLS6axQIgdgcUHFROQlgdEJK7GdKdXJA/FoHyHs1ixaH6ghQ29O0=',
+  'MIIBDTCBtKADAgECAhA6/8nwGPOoljnxmIMqqvoNMAoGCCqGSM49BAMCMAAwHhcNNzAwMTAxMDAwMDAwWhcNMjUxMTIyMDgyMjEyWjAAMDkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDIgAC3A9V8ynqRcVjADqlfpZ9X8mwbew0TuQldH/QOpkadsWjMDAuMCwGA1UdEQQlMCOCITJlM2YtODktMjA1LTEyOC0zNS5uZ3Jvay1mcmVlLmFwcDAKBggqhkjOPQQDAgNIADBFAiEAzhvfc2gHsvdOwSZ4R+XENjtmYN6oTE6e57Ibf7q3tKkCICBPHGL9K4LnHGgnoB/13ThVwaS5hCVU/kjqc3ixGMxq',
```
Comment (TimoGlastra) [inline apps/easypid/src/constants.ts:22]: what are these? can you add labels?

**Review by TimoGlastra — APPROVED** [review]

File: apps/easypid/src/constants.ts (hunk 2 above)
Comment (berendsliedrecht) [reply apps/easypid/src/constants.ts:22]: for testing, will be removed.

File: apps/easypid/src/constants.ts (hunk 1 above)
Comment (berendsliedrecht) [reply apps/easypid/src/constants.ts:7]: its PEM vs DER IIRC. can revert, does not matter.

**Pull request merged**